### DISCUSSION TITLE
Fork comment

### DISCUSSION THREAD

**Discussion by kapcod** [community/community#17181] [Other Feature Feedback, Questions, & Ideas]
People always fork projects for some reason. It could be "fix a bug", "switch to new version of X", "add this feature", etc.
Not always it makes sense to PR it back into original repo, not always original repo is being maintained and so some forks with really valuable changes can be there but nobody can find it.

It would be good to allow people to add fork comment/description and show it on the "forks" screen ideally together with more data like number of new commits (X commits ahead, Y commits behind) and last commit time. I see the list UI like this:

kapcod/some-project:  "Fix issue with integation with rails 7 and resque 2" ↑ 10 ↓ 7, forked 5 months ago, last commit 1 month ago.
Long, but would really help locating right fork.

**Comment by muir** [comment]
Please do this. 

What I'm really hoping for is a solution that allows a temporary maintainer to take over when a primary project isn't getting much attention. 

The fork comment would need to be updateable.   The forked repo would need to be able to have pull requests merged into it that would in turn become un-merged pull requests on the original repo.   If the original maintainer starts working again, the pending pull requests would allow the original repo to catch up.  In the meantime, the community of willing contributors would have a place to share work.
